connectors/presto/presto-snowflake/src/main/resources/checkstyle/presto-checks.xml

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE module PUBLIC
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd">
+<module name="Checker">
```

Comment:
Having a different set of checks per connector won't scale.  Any way we can just use the Airlift checks?